consensus/istanbul/validator.go

```diff
@@ -120,6 +133,9 @@ type ValidatorSet interface {
 	// Copy validator set
 	Copy() ValidatorSet
 
+	CacheUncompressed()
```

Comment:
Add Comments to both here and in Validator.

Also, rename `CacheUncompressed` to `CacheUncompressedBLSKey`